client/src/app/modules/material/material-module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { MatSidenavModule } from '@angular/material/sidenav';
+
+@NgModule({
+  declarations: [],
+  imports: [
+    MatSidenavModule,
+  ],
+  providers: [],
+  exports: [
+    MatSidenavModule,
+  ]
+})
+export class MatModule { }
```

Comment:
What is the reason behind creating this module?

Reply:
So as I mentioned in the conversation about the angular material module, what we can do from now on is add all of our angular material used inside this module, and then if we need those components in other particular modules such as auth-module and alta-main module we would just have to import this MatModule which will be centralizing the import of stuff related to angular material. What do you think about that?

Reply:
Yes exactly. That was my reasoning.